src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+@import "./mixin.scss";
+
+.calendar {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  flex-wrap: wrap;
+
+  margin: 100px auto;
+  padding: 10px;
+  gap: $gap;
+  width: $size-calendar;
+
+  & ::before {
+    display: block;
+    width: $cell-size;
+    align-items: center;
+    text-align: center;
+    font-size: 30px;
+    font-family: "Arial", sans-serif;
+    // position: absolute;
+  }
+
+  &__day {
+    @include square($cell-size);
+    background-color: #eee;
+    border: 1px solid black;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+    }
+
+    cursor: pointer;
+    transition: transform 0.5s;
```

Comment:
why do you have these styles after the :hover rule, move all the styles that are connected to the element before nesting rules